config/db_schemas/user.schema.js

```diff
@@ -15,7 +15,6 @@ const userSchema = new Schema(
     // This is the user's public display name
     displayName: {
         type: String,
-        required: true,
```

Comment:
This should not be removed because it is a requirement

Reply:
Yeah, yup that's true. idk why I removed that.